testing/e2e/app/routes/InformationPanel/spec.ts

```diff
@@ -0,0 +1,84 @@
+import { test, expect, type Page } from '@playwright/test';
+
+test.describe('Information Panel Resizing', () => {
+  test('should adjust information panel width', async ({ page }) => {
+    await page.goto('/InformationPanel');
+    const infoPanel = page.locator('#InformationPanel');
+    const initialWidth = (await infoPanel.boundingBox())!.width;
+
+    await resizeInformationPanel(page);
+
+    const newWidth = (await infoPanel.boundingBox())!.width;
+    expect(newWidth).toBe(initialWidth - 100);
+  });
+  test('should reset styles on orientation change', async ({ page }) => {
+    await page.goto('/InformationPanel');
+    const infoPanel = page.locator('#InformationPanel');
+    const infoPanelWrapper = page.locator('#InformationPanelWrapper');
+    const wrapperWidth = (await infoPanelWrapper.boundingBox())!.width;
+    const wrapperHeight = (await infoPanelWrapper.boundingBox())!.height;
+    const orientationButton = page.getByRole('button');
+
+    const initialWidth = (await infoPanel.boundingBox())!.width;
+
+    {
```

Comment:
can you add short comments before every block to indicate what the block is doing/testing?

Reply:
Sure.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1964/commits/ebc06196b0575adf72724056bcc9171ee8e18291)